hadoop/util.sh

```diff
@@ -73,3 +73,45 @@ function use_java8() {
    restart_worker_services
  fi
 }
+
+# Wait until namenode is out of safe mode. Takes 2 optional params
+# first : Number of attempts function will make to get namenode out of safemode. Default is 50
+# second : Number of seconds each attempt will sleep for waiting for namenode to come out of sleep mode. Default is 5 sec
+function wait_until_namenode_out_of_safe() {
+    n=0
+    num_attempts=$1
+    sleep_sec=$2
+    
+    if [ -z "$num_attempts" ]; then
```

Comment:
Nit - can change initialization to num_attempts=${1:-num_attempts} and similarly for sleep_sec.